docs/custom_virtualenvs.md

```diff
@@ -0,0 +1,71 @@
+Managing Custom Python Dependencies
+===================================
+awx installations pre-build a special [Python
+virtualenv](https://pypi.python.org/pypi/virtualenv) which is automatically
+activated for all `ansible-playbook` runs invoked by awx (for example, any time
+a Job Template is launched).  By default, this virtualenv is located at
+`/var/lib/awx/venv/ansible` on the file system.
+
+awx pre-installs a variety of third-party library/SDK support into this
+virtualenv for its integration points with a variety of cloud providers (such
+as EC2, OpenStack, Azure, etc...)
+
+Periodically, awx users want to add additional SDK support into this
+virtualenv; this documentation describes the supported way to do so.
+
+Preparing a New Custom Virtualenv
+=================================
+awx allows a _different_ virtualenv to be specified and used on Job Template
+runs.  To choose a custom virtualenv, first create one in `/var/lib/awx/venv`:
```

Comment:
I should probably adapt this instruction of `/var/lib/awx/venv` to be correct for dev+container environments, not production Tower installs.

Reply:
or possibly just point out that the venvs *could* be in different locations depending on the environment.

Reply:
Yea, on second thought, I think this is fine; `/venv/` is specific to the Docker-based development environment.